Source/WebCore/platform/gstreamer/VideoEncoderPrivateGStreamer.cpp

```diff
@@ -596,9 +611,32 @@ static void webkit_video_encoder_class_init(WebKitVideoEncoderClass* klass)
         }, [](GstElement*, LatencyMode) {
             notImplemented();
         });
+
+    auto setVpxEncoderInputFormat = [](auto* self) {
```

Comment:
A personal preference, not strong opinion: A function that does not capture anything and that is not used as a parameter for anything does not need to be a lambda, it can be a static above:

Reply:
I personally prefer this ;) I don't see the problem with capturing, imho it's safe in this specific context.